vaadinfrontend/frontend/themes/datamanager/components/dialog.css

```diff
@@ -18,6 +19,66 @@ vaadin-dialog-overlay::part(title) {
   margin-inline-start: 0;
 }
 
+/* set the width of the notification */
+.notification-dialog::part(overlay) {
+  width: 36.75rem;
+}
```

Comment:
NIT: IMO the width of the component should be dependent on the structure of the content instead of hardcoded in the css

Reply:
This value is taken from the design. It is the same for every notification dialogue.